framework/includes/option-types/addable-box/class-fw-option-type-addable-box.php

```diff
@@ -16,20 +16,26 @@ public function get_type()
 	 */
 	protected function _enqueue_static($id, $option, $data)
 	{
-		wp_enqueue_style(
-			'fw-option-'. $this->get_type(),
-			fw_get_framework_directory_uri('/includes/option-types/'. $this->get_type() .'/static/css/styles.css'),
-			array(),
-			fw()->manifest->get_version()
-		);
+		static $enqueue = true;
+
+		if($enqueue) {

```

Comment:
`if ($enqueue) {` Please

It is something minor but it hurts my eyes #perfectionist